### PR TITLE
xformers 20, and ui update

### PR DESCRIPTION
**Pull request by nawnie**

### CODE REVIEW

**Comment by nawnie** [comment]
Is rdy

File: Train_Colab.ipynb [hunk 1]
```diff
@@ -302,11 +383,12 @@
...
         "Dataset_Location = \"/content/drive/MyDrive/training_samples\" #@param {type:\"string\"}\n",
         "\n",
-        "model = save_name\n",
+        "if not resume:\n",
+        "  model = save_name\n",
         "\n",
         "#@markdown * Max Epochs to train for, this defines how many total times all your training data is used. Default of 100 is a good start if you are training ~30-40 images of one subject. If you have 100 images, you can reduce this to 40-50 and so forth.\n",
         "\n",
-        "Max_Epochs = 100 #@param {type:\"slider\", min:0, max:200, step:5}\n",
+        "Max_Epochs = 200 #@param {type:\"slider\", min:0, max:200, step:1}\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:391]: is there a reason this changed?  Comment says "default of 100".

File: Train_Colab.ipynb [hunk 2]
```diff
@@ -571,7 +663,10 @@
         "  \"disable_textenc_training\": False,\n",
         "  \"disable_xformers\": False,\n",
         "  \"disable_amp\": False,\n",
+        "  \"lowvram\": False,\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:666]: lowvram doesn't exist, this line should be removed

File: Train_Colab.ipynb [hunk 3]
```diff
@@ -571,7 +663,10 @@
...
+        "  \"notebook\": False,\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:667]: notebook doesn't exist, this line should be removed

File: Train_Colab.ipynb [hunk 4]
```diff
@@ -571,7 +663,10 @@
...
         "  \"save_optimizer\": False,\n",
+        "  \"scale_lr\": False,\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:669]: scale_lr doesn't exist, this should be removed

**Comment by victorchall** [comment]
Did you test training SD1.5 and SD2.1?  Any time something like xformers is updated I worry that one or the other could break.

At least test to first sample and see if sample looks ok.

File: Train_Colab.ipynb (hunk 1 above)
Comment (victorchall) [reply Train_Colab.ipynb:391]: I tend to think 100 is a better default than 200 as well.

**Comment by nawnie** [comment]
Oh wow I left those args in for awhile thank you.

I must of left it at 200 while testing.
So issue got brought up by user Q.
normally broken xformers either gives us blank or doesn't train, 19 did
train with 2.01 but the loss was horrobly inconsistent.

Upon researching I found a post on the xformer github stating 2.01 is NOT
supported in 19 but will be in 20 later this week.

Testing samples came out well but had to do a full test not just to first
sample to make sure it was training well. 😅 I'll get that fixed up to you
tonight ty for the reply.

**Comment by victorchall** [comment]
"Loss horribly inconsistent" how?  

If you have a known "good" dataset it helps to confirm loss/val looks ok with normal settings.  I usually train just one character as a test, a character/set I know works normally at default settings.

**Comment by nawnie** [comment]
I can post pics of his logs in a dm to you

**Pull request closed without merging**